website/data/docs/guides/integrating-with-payment-services.mdx

```diff
@@ -42,18 +42,18 @@ const response = await client.charges.create({
 
 ## Integrating with Paypal
 
-The [Paypal](https://www.paypal.com/) payment platform provides APIs to process payments and manage orders. Unlike most platforms, it expects a money representation with an amount in major currency units. You can use [`toUnit`](/docs/api/formatting/to-unit) to retrieve this value.
+The [Paypal](https://www.paypal.com/) payment platform provides APIs to process payments and manage orders. Unlike most platforms, it expects a money representation with an amount in major currency units. You can use [`toDecimal`](/docs/api/formatting/to-decimal) to retrieve this value.
```

Comment:
Hmm, "retrieve" works better here I believe because the user's goal is to "get" the value, not to format.

Reply:
Sounds good, my logic was that `toDecimal` is one of the format functions.

Reply:
Maybe this could be rewritten into:

```suggestion
The [Paypal](https://www.paypal.com/) payment platform provides APIs to process payments and manage orders. Unlike most platforms, it expects a string representation with an amount in major currency units. You can use [`toDecimal`](/docs/api/formatting/to-decimal) to format the object and pass this value to Paypal.
```

Reply:
Very nice.